### PR TITLE
ButtonIcon: Add ledger.

### PR DESCRIPTION
**Pull request by JoeGruffins**
It appears I need the ledger tag added there to Button Icons in order to import to decrediton... Also unsure how to use the exported settings. It appears that the view box needs to be set to those values? I am an idiot here though so please help me correct but this is working. @amass01

### CODE REVIEW

**Review by amass01 — APPROVED** [review]

File: src/components/Icon/Icon.jsx [hunk 1]
```diff
@@ -1402,11 +1402,12 @@ export const IconWrapper = React.forwardRef(
     const defaultIconBgColor =
       backgroundColor || getThemeProperty(theme, "icon-background-color");
     const isStringSize = typeof size === "string";
+    const vb = type === "ledger" ? "0 0 148 128" : viewBox;
```
Comment (amass01) [inline src/components/Icon/Icon.jsx:1405]: can you pass this instead of hardcoding ?

**Review by amass01 — CHANGES_REQUESTED** [review]

File: src/components/Icon/Icon.jsx (hunk 1 above)
Comment (JoeGruffins) [reply src/components/Icon/Icon.jsx:1405]: Is the viewBox not always the same for the same full svg? I don't understand %100 but If we just want the ledger logo, we want this, if we want to change the size we would change the size...

Comment (JoeGruffins) [reply src/components/Icon/Icon.jsx:1405]: I need to find a way to show this live...

Comment (JoeGruffins) [reply src/components/Icon/Icon.jsx:1405]: https://liveweave.com/Z5rJgM

So, can play with that, but using and view box but that one will just make a bunch of white space you can't control or not show the logo at all it seems.

Comment (amass01) [reply src/components/Icon/Icon.jsx:1405]: hrmmm then i think we may need a designer to tweak the svg viewbow to match the default we use for all other svgs

Comment (amass01) [reply src/components/Icon/Icon.jsx:1405]: @tiagoalvesdulce i know that you did that in the past pretty straight forward, would like to get 
your help/opinion

Comment (JoeGruffins) [reply src/components/Icon/Icon.jsx:1405]: Yeah, changing the viewbox to our default would also be a good answer. I will try some simple things to see if I can (load and save the svg with gimp/imagemagick).

Comment (JoeGruffins) [reply src/components/Icon/Icon.jsx]: Last change changes the original to work with the "0 0 450 450" viewbox https://github.com/decred/pi-ui/compare/b00ad6092b255870079cf2cb03175ef3be8412fc..af3dfbe00bc6f7d85a9ad61a6d42641792a0575c

Created by taking the svg from here https://developers.ledger.com/docs/transport/ledger-logos/ and the following transformation:
```
rsvg-convert ledger-square.svg -a -w 450 --page-width 450 --page-height 450 --top 29 -f svg -o ledger450.svg
```

Comment (amass01) [reply src/components/Icon/Icon.jsx]: awesome!!!!

**Review by amass01 — APPROVED** [review]

File: src/components/Icon/Icon.jsx (hunk 1 above)
Comment (tiagoalvesdulce) [reply src/components/Icon/Icon.jsx]: sorry for the late response. Yeah, I used to use Sketch to adjust the viewbox.

**Review by tiagoalvesdulce — APPROVED** [review]
LGTM

**Pull request merged**